public/components/overview/office-panel/config/main-view-config.tsx

```diff
@@ -49,5 +50,37 @@ export const MainViewConfig = {
         },
       ],
     },
+    {
+      columns: [
+        {
+          width: 50,
+          component: (props) => (
+            <EuiFlexItem grow={props.grow}>
+              <AggTable
+                tableTitle={'Top rules'}
+                aggTerm={'rule.description'}
+                aggLabel={'Rule'}
+                maxRows={'5'}
```

Comment:
**thought**: For the string value case, we could pass the value directly without interpreting JS. It works anyways, it is only a personal preference.

Reply:
Commited